R/birdlife.R

```diff
@@ -25,3 +25,46 @@ birdlife_habitat = function(id){
   out = cbind(id, tables[[habitat_table_number]])
   out[-nrow(out), ] # Drop last row (altitude)
 }
+
+
```

Comment:
Cool, need to associate it with a man file though...currently, nothing happens when you execute `?birdlife_threats`, maybe #' @rdname birdlife_habitat